alchemiscale/tests/unit/test_storage_models.py

```diff
@@ -137,40 +137,40 @@ def test_from_dict(self):
         assert trp_reconstructed.taskhub_scoped_key == original_taskhub_scoped_key
 
 
-class TestTraceback(object):
+class TestTracebacks(object):
```

Comment:
These tests don't currently pass, since we don't pass in `source_keys` or `failure_keys` in `Tracebacks` upon init.